src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -969,52 +972,35 @@ void readAnalyze(TrackPointer track, double sampleRate, int timingOffset, bool i
+            memory_cue_loop_t memoryCueOrLoop = memoryCuesAndLoops[memoryCueOrLoopIndex];
 
-            if (memoryCueLoop.endPosition == -1) {
+            if (memoryCueOrLoop.endPosition == Cue::kNoPosition) {
                 if (mainCueFound) {
                     // Add remaining memory cues as hot cues (after actual found hotcues) as Mixxx can only have 1 cue
-                    setHotCue(track, memoryCueLoop.startPosition, -1, ++lastHotCueIndex, memoryCueLoop.comment, mixxx::RgbColor::nullopt(), memoryCueLoop.colorID);
+                    setHotCue(track, memoryCueOrLoop.startPosition, -1, ++lastHotCueIndex, memoryCueOrLoop.comment, mixxx::RgbColor::nullopt(), memoryCueOrLoop.colorID);
                 } else {
                     // Set first chronological memory cue as Mixxx MainCue
-                    track->setCuePoint(CuePosition(memoryCueLoop.startPosition));
+                    track->setCuePoint(CuePosition(memoryCueOrLoop.startPosition));
                     CuePointer pMainCue = track->findCueByType(mixxx::CueType::MainCue);
-                    pMainCue->setLabel(memoryCueLoop.comment);
-                    if (memoryCueLoop.colorID != -1) {
-                        pMainCue->setColor(colorFromID(memoryCueLoop.colorID));
+                    pMainCue->setLabel(memoryCueOrLoop.comment);
+                    if (memoryCueOrLoop.colorID != kNoColorID) {
+                        pMainCue->setColor(colorFromID(memoryCueOrLoop.colorID));
                     }
                     mainCueFound = true;
                 }
             } else {
-                if (firstLoopFound) {
-                    // Add remaining loops as hot cues (after actual found hotcues) as Mixxx can only have 1 loop
-                    // Mixxx v2.4 will feature multiple loops, so these saved here will be usable 
-                    setHotCue(track, memoryCueLoop.startPosition, memoryCueLoop.endPosition, ++lastHotCueIndex, memoryCueLoop.comment, mixxx::RgbColor::nullopt(), memoryCueLoop.colorID);
-                } else {
-                    // Set first chronological loop as Mixxx loop
-                    CuePointer pLoop = track->findCueByType(mixxx::CueType::Loop);
-                    if (!pLoop) {
-                        pLoop = CuePointer(track->createAndAddCue());
-                    }
-                    pLoop->setStartPosition(memoryCueLoop.startPosition);
-                    pLoop->setEndPosition(memoryCueLoop.endPosition);
-                    pLoop->setLabel(memoryCueLoop.comment);
-                    if (memoryCueLoop.colorID != -1) {
-                        pLoop->setColor(colorFromID(memoryCueLoop.colorID));
-                    }
-                    pLoop->setType(mixxx::CueType::Loop);
-                    firstLoopFound = true;
-                }
+                // Add remaining loops as hot cues (after actual found hotcues) as Mixxx can only have 1 loop
```

Comment:
This line is no longer accurate, right? I think we can just remove the whole line.